# HG changeset patch
# User aph
# Date 1393513709 0
#      Thu Feb 27 15:08:29 2014 +0000
# Node ID 04e4c3ec6516727f01f91a9ce8cb72586a3bc502
# Parent  942d4ba93be74b1c401d6532f116da80f5466303
OPENJDK6-29: JDK fails to zero jdk_version_info correctly
Reviewed-by: andrew

diff -r 942d4ba93be7 -r 04e4c3ec6516 src/share/native/common/jdk_util.c
--- openjdk/jdk/src/share/native/common/jdk_util.c	Wed Feb 26 18:06:02 2014 +0000
+++ openjdk/jdk/src/share/native/common/jdk_util.c	Thu Feb 27 15:08:29 2014 +0000
@@ -76,7 +76,7 @@
     }
 
 
-    memset(info, 0, sizeof(info_size));
+    memset(info, 0, info_size);
     info->jdk_version = ((jdk_major_version & 0xFF) << 24) |
                         ((jdk_minor_version & 0xFF) << 16) |
                         ((jdk_micro_version & 0xFF) << 8)  |
